client/core/core.go

```diff
@@ -1954,6 +2011,11 @@ func (c *Core) ReconfigureWallet(appPW, newWalletPW []byte, assetID uint32, cfg
 		EncryptedPW: oldWallet.encPW(),
 		Address:     oldDepositAddr,
 	}
+	seeded := oldWallet.Info().Seeded
+	if seeded && len(newWalletPW) > 0 {
+		return newError(passwordErr, "cannot set a password on a seeded wallet")
+	}
```

Comment:
Can move this to before the `oldDepositAddr := oldWallet.currentDepositAddress()` line. Prefer to perform checks like these early on before creating/assigning variables that will not be used if the check fails.